docs/src/cookbooks/cookbook.md

```diff
@@ -462,3 +464,52 @@ ChiselStage.emitVerilog(new Salt)
 ```scala mdoc:verilog
 ChiselStage.emitVerilog(new Salt)
 ```
+
+## Directionality
+
+### How do I strip directions from a bidirectional Bundle (or other Data)?
+
+Given a bidirectional port like a `Decoupled`, you will get an error if you try to connect it directly
+to a register:
+
+```scala mdoc:silent
+import chisel3.util.Decoupled
+class BadRegConnect extends Module {
+  val io = IO(new Bundle {
+    val enq = Decoupled(UInt(8.W))
+  })
+  
+  val monitor = RegNext(io.enq)
+}
+```
+
+```scala mdoc:crash
+ChiselStage.emitVerilog(new BadRegConnect)
+```
+
+While there is no construct to "strip direction" in Chisel3, wrapping a type in `Output(...)` will
+coerce the direction to `Output` (even if the input is bidirectional).
+This will have the desired result when used to construct a Register:
+
+```scala mdoc:silent
+import chisel3.util.Decoupled
+class CoercedRegConnect extends Module {
+  val io = IO(new Bundle {
+    val enq = Flipped(Decoupled(UInt(8.W)))
+  })
+  
+  // Make a Reg of the correct type instead of using RegNext
+  val monitor = Reg(Output(chiselTypeOf(io.enq)))
+  monitor := io.enq // monoconnect will ignore directions of right-hand side
```

Comment:
```suggestion
  monitor := io.enq // monoconnect will ignore directions of right-hand side
```

This comment is confusing. *why* will monoconnect ignore the directions. Is it because monoconnect always ignores it? Or are you saying "monoconnect always looks to the LHS for the directions, and in case they are all outputs / Sinks, therefore we will grab everything from io.enq regardless of the source/sink directionality of io.enq"

Reply:
Would "monoconnect ignores directions" be better? Or no comment?

Reply:
Well, why does it have to be `Output` instead of `Input`?

Reply:
Because `Output` is the default direction 🙃

Reply:
I take it back, `Input(...)` works fine too: https://scastie.scala-lang.org/4BrQKZAoT8S9x4kxDcGNlQ

Reply:
🤯

Reply:
this is making me feel like we should create a `StripDirection` alias for `Output`

Reply:
I guess I still don't get the monoconnect rules in general given this conversation.

Reply:
`<>` will see that the RHS is bidirectional and use that instead of just having things flow from right to left.

Reply:
ok, then maybe just "monoconnect ignores directions from the right hand side" if that is accurate statement...